### ISSUE TITLE
Missing llvm-ar ?

### ISSUE THREAD

**Issue by mattgodbolt** [grailbio/bazel-toolchain#26]
I just updated to 2200d5315ddd3071985a2e82e574da525617867e and now I get:

```
src/main/tools/linux-sandbox-pid1.cc:427: "execvp(external/llvm_toolchain/bin/llvm-ar, 0xbf08a0)": No such file or directory
```

when building. The most recent commit appears to have been something that tried to fix this (making a guess) - any ideas? (LLVM 8.0.0 but 9.0.0 fails too, on Linux 64)

**Comment by mattgodbolt** [comment]
```
~/d/core (master|✔) $ bazel-core/external/llvm_toolchain/bin/llvm-ar --version
LLVM (http://llvm.org/):
  LLVM version 8.0.0
  Optimized build.
  Default target: x86_64-unknown-linux-gnu
  Host CPU: skylake-avx512
```

**Comment by mattgodbolt** [comment]
Running with sanbox debugging and then `ls`-ing the directory that actually gets mounted in the sandbox:

```
~/d/core (master|✔) $ ls /home/mgodbolt/.cache/bazel/_bazel_mgodbolt/d5bad8eff3da01980a0963475d1edfc1/sandbox/linux-sandbox/2019/execroot/__main__/external/
bazel_tools/  llvm_toolchain/
~/d/core (master|✔) $ ls /home/mgodbolt/.cache/bazel/_bazel_mgodbolt/d5bad8eff3da01980a0963475d1edfc1/sandbox/linux-sandbox/2019/execroot/__main__/external/llvm_toolchain/bin/
clang@  clang++@  clang-cpp@  ld@  ld.gold@  ld.lld@
```

**Comment by mattgodbolt** [comment]
I just did a binary search of revisions and found that the revision 2200d53 is the culprit. Everything works if I stick to 8f2721aaf4db902529eb92cdf078d4e68dbd4173

**Comment by siddharthab** [comment]
I think that commit put a minimum bazel version requirement of 1.0. Does it work for you on that?

**Comment by mattgodbolt** [comment]
At the moment I'm blocked from using bazel 1.0 by other upstream dependencies failing (mostly grpc-related). I'll try when I can get those updated and working. Thanks!

**Comment by siddharthab** [comment]
Thank you. I will add the minimum version requirement so the messaging is clear.

The unresolved issues in this project will require bazel 1.0, so there is reason for us to support only versions 1.0 and up.

**Comment by mattgodbolt** [comment]
Sounds good to me: thank you!

**Comment by mattgodbolt** [comment]
Confirmed fixed if I upgrade to 1.0: thanks!
